frontend/src/common/components/tabs/TabsList.scss

```diff
@@ -26,13 +26,13 @@
         opacity: 1;
         padding: 0.875rem 0;
       }
-  
-      &__count {
-        font-size: 0.875rem;
-        color: $bc-black;
-        background-color: $bc-border-grey;
-        margin-left: 1rem;
-      }
+      // TODO remove this if we no longer need tab counters
+      // &__count {
```

Comment:
Technically, if we are not using counters, the element with ```&__count``` wouldn't appear, so including this style rule here wouldn't be a problem (just that it wouldn't have any effects)